src/AdyenPaymentShopware6.php

```diff
@@ -267,6 +265,11 @@ private function updateTo160(UpdateContext $updateContext): void
             );
         }
     }
+
+    private function updateTo200(UpdateContext $updateContext): void
+    {
+        // todo add amazon pay to the next release version
```

Comment:
I think we can add this now right? Why would you like to wait?

Reply:
You're right, no reason to wait for the release PR